setup.py

```diff
@@ -1,11 +1,7 @@
 import os
-import sys
-
-python_path = sys.executable
-requirements_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "requirements.txt")
 
 print("Checking Dependencies...")
-os.system("{0} -m pip install -q -r {1}".format(python_path, requirements_path))
+os.system("python -m pip install -q -r requirements.txt")

```

Comment:
should've maintained history. avoid force pushes for commit names which specifies exactly the opposite. In this case, commit message says "absolute python path added".